sass/bhclub.scss

```diff
@@ -113,12 +124,73 @@ nav {
 
 // Main Content
 main {
-    padding: 3.5rem;
+    padding: 1.75rem 3.5rem;
     padding-left: calc(25% + 3.5rem); // pushes it outside of the nav while allowing for backgrounds to stretch underneath
 }
 
 section {
-    padding: 5rem 0;
+    padding: 1.75rem 0;
+    min-height: 100vh;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+}
+
+// About Section
+#about p {
+    width: 60%;
+}
+
+.card {
+    border: 2px solid $blue;
+    border-radius: 6px;
+    margin-top: 2rem;
+    width: 75%;
+
+    .card-header {
+        font-weight: 600;
+        color: $blue;
+        text-align: left;
+        font-size: 1.75rem;
+        background-color: $off-white;
+        border: none;
+        width: 100%;
+        padding: 0.625rem 1.25rem;
+        cursor: pointer;
+        transition: all 1s;
+
+        .fa {
+            float: right;
+            color: $blue;
+        }
+
+        &:hover {
+            background-color: $blue;
+            color: $off-white;
+
+            .fa {
+                color: $off-white;
+            }
+        }
+    }
+
+    .card-body {
+        padding: 1.875rem 1.25rem;
+        display: none;
+    }
+}
+
+.card-open {
+    .card-header {
+        background-color: $dark-blue;
+        color: $off-white;
+        border-bottom: 2px solid $blue;
+        border-radius: 0px 0px 6px 6px;
```

Comment:
does border radius not follow in the footsteps of margin for `horizontal vertical;`?

Reply:
I tried, because it's the corners it goes from the top left clockwise. 0px 6px makes the top left and bottom right sharp while the other two are rounded.

Reply:
gotcha!